docs/_index.md

```diff
@@ -282,42 +282,68 @@ To uninstall Poetry, simply delete the entire `$VENV_PATH` directory.
 `poetry` supports generating completion scripts for Bash, Fish, and Zsh.
 See `poetry help completions` for full details, but the gist is as simple as using one of the following:
 
+### Bash
+
+There are few options in `bash`, choose the one that suits your workflow.
```

Comment:
I think we can drop this line as well if we go with my simplification.

Reply:
sure, removed it.